Bottom: f38ce59333518ddbd20919f0e5b2953edf3716c1
Top:    2d7d896cee2e25449805245dc88096ac38cec3e9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-24 00:45:33 +0000

find.c: Pass flags to sub-functions without using static variable "TheFlag".


---

diff --git a/src/find.c b/src/find.c
index 8f4434e..e96dc99 100644
--- a/src/find.c
+++ b/src/find.c
@@ -268,7 +268,6 @@ typedef struct
  * some local identifiers
  */
 static Coord Bloat = 0;
-static int TheFlag = FOUNDFLAG;
 static void *thing_ptr1, *thing_ptr2, *thing_ptr3;
 static int thing_type;
 static bool User = false;    /* user action causing this */
